content/gocourse-2019-05-melbourne.slide

```diff
@@ -482,24 +520,29 @@ Document them
 
 * Lab setup
 
-- Clone [[https://github.com/anz-bank/go-course][https://github.com/anz-bank/go-course]]
-- Follow instructions on [[https://github.com/anz-bank/go-course/blob/master/README.md][README.md]]
+* Pre-requites
+
+- Install [[https://git-scm.com/downloads][git]]
+- Install [[https://golang.org/doc/install][go 1.12]]
+- Install [[https://github.com/golangci/golangci-lint#install][golangci-lint]]
 
-As part of all labs
 
-- Test your code
-- Create a PR following the [[https://github.com/anz-bank/go-course#pre-pr-checklist][PR checklist]]
+* Repository setup overview
 
+Clone your fork of [[https://github.com/anz-bank/go-course][anz-bank/go-course]]
 
-* PR from own fork - initial setup
+Issue PRs
```

Comment:
"Open PRs"? "Create PRs"? This is a little bit ambiguous without the sentence that was there (issue is a noun and a verb, and a github-relevant noun since PRs are actually issues under the hood)

Reply:
fair point, done.